### PR TITLE
add libmagic

### PR DESCRIPTION
**Pull request by mvdbeek**
Is required for complete functionality of sra-toolkit [building of ncbi-copycat] (PR #598).

### CODE REVIEW

File: packages/package_libmagic_5_25/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="libmagic" version="5.25">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url" sha256sum="3735381563f69fb4239470b8c51b876a80425348b8285a7cded8b61d6b890eca">ftp://ftp.astron.com/pub/file/file-5.25.tar.gz</action>
```
Comment (erasche) [inline packages/package_libmagic_5_25/tool_dependencies.xml]: cargo port?

Comment (mvdbeek) [inline packages/package_libmagic_5_25/tool_dependencies.xml]: let' see. I don't think we actually need it. but if we do, I'll do a PR to cargo.

Comment (mvdbeek) [inline packages/package_libmagic_5_25/tool_dependencies.xml]: @erasche I guess the url stays the same, if I set upstream in cargo, right?

Comment (erasche) [inline packages/package_libmagic_5_25/tool_dependencies.xml]: @mvdbeek yep, people weren't fond of downloading from cargo port. Maybe we'll have @bgruening's download_from_proxy action eventually and will be able to use the cargo port.

Comment (bgruening) [inline packages/package_libmagic_5_25/tool_dependencies.xml]: @erasche do you think I should still work on this or is conda the way forward?

Comment (erasche) [inline packages/package_libmagic_5_25/tool_dependencies.xml]: @bgruening argh. Right, conda. Nope, you're right, conda is the way forward.

**Comment by bgruening** [comment]
Thanks @mvdbeek !

**Pull request merged**